app/main.py

```diff
@@ -1,8 +1,60 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            comfort_class: int,
+            clean_mark: int,
+            brand: str
+    ) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            distance_from_city_center: float = 1,
+            clean_power: int = 0,
+            average_rating: float = 0.0,
+            count_of_ratings: int = 0
+    ) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list) -> float:
+        filtered_cars = [
+            car
+            for car in cars
+            if car.clean_mark <= self.clean_power
+        ]
+        income = sum(
+            [
+                self.calculate_washing_price(car)
+                for car in filtered_cars
+            ]
+        )
+        for car in filtered_cars:
+            self.wash_single_car(car)
+        return round(income, 1)
```

Comment:
Consider breaking down the `serve_cars` method into smaller, more focused methods for improved readability